### DISCUSSION TITLE
Is it possible to pass in precomputed word timings?

### DISCUSSION THREAD

**Discussion by laphang** [pyannote/pyannote-audio#1058] [Q&A] [answered]
I have an use case where I want to do speech recognition and then diarization, and I've been exploring using pyannote for the latter and seeing good accuracy. 

I was wondering whether it's possible to pass in the word timings already obtained from speech recognition into pyannote (ie skip the VAD step / improve speed).

**Comment by hbredin** [comment, marked as answer]
Not out of the box. Also, the speaker diarization pipeline does not rely on a VAD step _per se_.
So you would be better off rewriting a pipeline from scratch.

I am also curious about your use case and the "_good accuracy_" you got.  
Feel free to drop me an email to tell me a bit more :)
